tests/installation/addon_products_sle.pm

```diff
@@ -197,28 +198,32 @@ sub run {
                 send_key 'alt-o';                                                   # continue
             }
             handle_addon($addon);
+            # add another add-on if $addon is not first from all ADDONS and not in SLE 15+
+            send_key 'alt-a' if (is_sle('<15') && ((split(/,/, get_var('ADDONS')))[-1] ne $addon));
+        }
+        if (is_sle('15+') && $addon !~ /^ses$|^rt$/) {
```

Comment:
If with this condition migration test will assert this screen, while we shouldn't do it.

Please change it and run verify run with https://openqa.suse.de/tests/3684503#

Reply:
I guess checking the value of `$addon` outside of the loop doesn't make sense, as it will be undefined.
Following code snippet:
```perl
my $addon;

for $addon (split(/,/, "1,2,3")) {
    say $addon;
}

say $addon;
```
Will put:
```
1
2
3

Use of uninitialized value $addon in say at ...
```

Reply:
Yes, not only that, it was also not allowed per perlcritic rules. I introduced a new variable for this.